src/features/game/lib/mapMovement.ts

```diff
@@ -0,0 +1,93 @@
+type MovementIntervals = {
+  moveUp?: ReturnType<typeof setInterval>,
+  moveDown?: ReturnType<typeof setInterval>,
+  moveLeft?: ReturnType<typeof setInterval>,
+  moveRight?: ReturnType<typeof setInterval>
+}
+
+let container: HTMLElement
+const sensitivity = 3
+const intervalTime = 1
+const movementIntervals: MovementIntervals = {}
+const initialCoordinates = [1024,1214]
+// TODO dynamic initial coordinates based on tile dimensions
+
+const keyDownListener = (event: KeyboardEvent) => {
+  const key = event.key.toLowerCase()
+  if (container) {
+    if (key === 'w' || key === 'arrowup') {
+      if (movementIntervals.moveUp == undefined) {
```

Comment:
Can we update this to use `===`? It's good practice to always use strict equality in JS.